exo-run/features/steps/steps-given.ls

```diff
@@ -18,4 +19,4 @@ module.exports = ->
                                        cwd: @app-dir,
                                        stdout: dim-console.process.stdout
                                        stderr: dim-console.process.stderr)
-      done!
+      wait 10_000, done
```

Comment:
Since we're using `docker-compose` to boot up our services, we can't individually monitor each service in its own ObservableProcess, meaning we can't wait for each of them to come online and then finally emit a message such as 'all services online'. That control is something we hand over when switching to `docker-compose`. 

I'm not sure how this step definition was working previously when we weren't waiting for any sort of online text. Regardless, that's why we're waiting for 10 seconds instead of for some sort of online text to ensure that an application has come online.